src/System.Management.Automation/engine/CommandDiscovery.cs

```diff
@@ -306,7 +306,7 @@ internal static void VerifyRequiredModules(ExternalScriptInfo scriptInfo, Execut
                         ScriptRequiresException scriptRequiresException =
                             new ScriptRequiresException(
                                 scriptInfo.Name,
-                                new Collection<string> { requiredModule.Name },
+                                new Collection<string> { requiredModule.ToString() },
```

Comment:
Should create a custom resx string here to make it more user friendly.

Reply:
@SteveL-MSFT I have added resx strings for the messages in 74451279f5f8341fd500116ee3828a6c89043e0c. Let me know what you think.